app/templates/components/forms/reset-password-form.hbs

```diff
@@ -21,6 +21,11 @@
               <div class="ui left icon input">
                 <i class="key icon"></i>
                 {{input type='password' name='password' placeholder=(t 'password') value=password}}
+                  <div class="ui small basic icon buttons">
+                    <button type="button" class="ui button" {{action 'showPassword' 'New'}}>
+                      <i class="{{if showPassNew 'hide' 'unhide'}} basic icon"></i>
+                    </button>
+                  </div>
```

Comment:
fix your CI by correcting indentation here.

Reply:
at line 24!

Reply:
yes

Reply:
Should I push commit now? because still unable to see changes??

Reply:
Share screenshots/GIF fir current behaviour .

Reply:
#3876

Reply:
It's still the same even after your commit?

Reply:
The validation is working fine . 

![Screenshot from 2020-01-28 20-40-26](https://user-images.githubusercontent.com/46647141/73276385-9d0be800-420e-11ea-9d49-4aea2aca1f0e.png)

Reply:
But I am unable to see the changes in my local machine

Reply:
see [ this comment](https://github.com/fossasia/open-event-frontend/pull/3882#issuecomment-579294124)